spec.md

```diff
@@ -514,6 +529,100 @@ int("0xffff", 16)               # 65535
 ```
 
 
+### Floating-point numbers
+
+The Starlark floating-point data type represents an IEEE 754
+double-precision floating-point number.
+Its [type](#type) is `"float"`.
+
+Arithmetic on floats using the `+`, `-`, `*`, `/`, `//`, and `%`
```

Comment:
Mention that arithmetic on floats can overflow the magnitude, resulting in an error. (Note that Starlark's lack of error handling makes it pretty unsuitable for intensive floating point computations, even if we didn't have the NaN issues.)

Reply:
This is not true: float x float arithmetic always yields a float, not an error (except division by zero, as mentioned).